packages/offix-datastore/tests/DataStore.test.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import "fake-indexeddb/auto";
+
+import { configure, save, query } from "../src/DataStore";
+import { createDefaultStorage } from "../src/storage";
+
+function getIndexedDB() {
+    return new Promise<IDBDatabase>((resolve, reject) => {
+        const request = window.indexedDB.open("offix-datastore", 1);
+        request.onerror = () => reject(request.error);
+        request.onsuccess = () => resolve(request.result);
+    });
+}
+
+function convertToPromise(request: IDBRequest) {
+    return new Promise((resolve, reject) => {
+        request.onsuccess = (event) => {
+            resolve(request.result);
+        }
+
+        request.onerror = (event) => {
+            reject(request.error);
+        }
+    });
+}
+
+beforeEach(() => {
+    configure(`${__dirname}/mock.graphql`);
+});
+
+afterEach(async () => {
+    await new Promise((resolve, reject) => {
+        indexedDB.deleteDatabase("offix-datastore").onsuccess = event => resolve();
+    });
+});
+
+test("Setup client db with provided schema", async () => {
```

Comment:
@Eunovo This test is failing for me locally. It looks like the test is timing out because the indexedDB.deleteDatabase function in the afterEach cycle is getting blocked and never resolving the promise.

I'm not sure if it will be the same in circleci becaues the pipeline failed before it got to the tests.

Do you mind fixing the lint issues so we can see if the tests run okay on the pipeline.

Are the tests all passing for you?

Reply:
I think the same is happening for the second test too

Reply:
All right

Reply:
The first test in `DataStore.test.ts` times out for me